sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/version.hpp

```diff
@@ -14,7 +14,7 @@
 #define AZURE_STORAGE_BLOBS_VERSION_MAJOR 12
 #define AZURE_STORAGE_BLOBS_VERSION_MINOR 0
 #define AZURE_STORAGE_BLOBS_VERSION_PATCH 0
-#define AZURE_STORAGE_BLOBS_VERSION_PRERELEASE "beta.4"
```

Comment:
Btw, if this follows the expected version.hpp syntax, like the other modules (core/identity), you will get auto-version increment update PRs that will do this for you, as part of the release.

For example:
https://github.com/Azure/azure-sdk-for-cpp/pull/951

Look out for that post-release. If it doesn't work, we would want to investigate.